### DISCUSSION TITLE
how to skip the texture

### DISCUSSION THREAD

**Discussion by pramodwerea** [zeux/meshoptimizer#421] [General]
Hi I would like to convert obj to GLB but without texture. what is the command to skip the texture? Thanks a lot.
Image in the folder there are texture file and obj but I would like to skip texture when generating GLB file.

**Comment by zeux** [comment]
There's no command line option like this atm. However, if the .mtl file is not found, the texture bindings are automatically skipped. So you should be able to remove the .mtl file to get geometry-only conversion if I understand the question correctly. If that doesn't help maybe you can describe your usecase in more detail?
